python/pyspark/sql/pandas/serializers.py

```diff
@@ -154,6 +155,9 @@ def create_array(s, t):
             # Ensure timestamp series are in expected form for Spark internal representation
             if t is not None and pa.types.is_timestamp(t):
                 s = _check_series_convert_timestamps_internal(s, self._timezone)
+            elif type(s.dtype) == pd.CategoricalDtype:
+                # FIXME: This can be removed once minimum pyarrow version is >= 0.16.1
```

Comment:
please change `FIXME` -> `NOTE`. It sounds like we are adding broken code, which isn't the case. It's just not needed after a certain version.

Reply:
Done